crates/fj-kernel/src/validation/mod.rs

```diff
@@ -28,15 +28,14 @@
 mod coherence;
 mod uniqueness;
 
-pub use self::{
-    coherence::{CoherenceIssues, CoherenceMismatch},
-    uniqueness::UniquenessIssues,
-};
-
 use std::{collections::HashSet, ops::Deref};
 
 use fj_math::Scalar;
 
+pub use self::{
+    coherence::{CoherenceIssues, CoherenceMismatch},
+    uniqueness::UniquenessIssues,
+};
```

Comment:
Could you please revert this change? Placing those `pub use` statements right below the module definitions is quite deliberate, as they are closely related to them, and are part part of the module's public interface, which makes them distinct from the other `use` statements. I believe that mixing `use` and `pub use` makes it harder to follow what's going on.

Reply:
Sorry, my bad. Seems I had rust-analyser configured to auto-format with some non-default options. Reverted.

Reply:
Thanks! :+1: 

If there's any way to better support your setup and make things easier for you, I'd be happy to merge any repository-specific configuration that helps with that. We already have VS Code and rustfmt config, as well as some others.

Reply:
Thanks, but it was definitely just a problem on my end. 👍